fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerMutualTest.java

```diff
@@ -440,7 +448,8 @@ void testMutualIndexingCrashAndAllowContinueNonMutually(boolean explicit) {
             try (OnlineIndexer indexBuilder = newIndexerBuilder()
                     .setIndex(index)
                     .setTimer(timer)
-                    .setIndexingPolicy(mutualTakeOverIndexingPolicy(explicit))
+                    .setIndexingPolicy(mutualTakeOverIndexingPolicy(explicit, false)
+                            .allowTakeoverContinue(true))
```

Comment:
doesn't `allowTakeover` here override the usage of `explicit` in the line above?
Wouldn't this be the same as:
```
                    .setIndexingPolicy(mutualTakeOverIndexingPolicy(false, false))
```

But I think the original version was intentional, to make sure that both `allowTakeoverContinue(true)` and `allowTakeoverContinue(List.of(...))` work the same.

Reply:
It is a leftover from a code change that was undone (but not fully undone - sorry). I removed line 452. 
✅